__tests__/app.test.js

```diff
@@ -0,0 +1,27 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+const app = require("../app");
+const request = require("supertest");
+
+beforeEach(() => {
+  return seed(data);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/topics", () => {
+  test("GET: 200, responds with an array containing all topics", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.length).not.toBe(0);
```

Comment:
this is okay but I recommend being specific on what the number is you know how many topics you have in your seeding data